src/main/java/bot/telegrambot/dao/TimeDao.java

```diff
@@ -0,0 +1,8 @@
+package bot.telegrambot.dao;
+
+import org.telegram.telegrambots.meta.api.methods.send.SendMessage;
+import org.telegram.telegrambots.meta.api.objects.Message;
+
+public interface TimeDao {

```

Comment:
```suggestion
public interface TimeService {
```
тут ми не працюємо з бд, тому скрізь заміни дао на сервіс